operating/external-ips.adoc

```diff
@@ -0,0 +1,234 @@
+
+In a nutshell:
+[cols="1,4,1,1", options="header"]
+|===
+|
+|Prerequisites
+|External IP API
+|External IP in Network Graph
+
+|Secured cluster
+|Enable external IPs collection per secured cluster runtime configuration
+|Yes
+|Yes
+
+.2+|Central services
+|Enable external IPs storage in Central via env variable: ROX_EXTERNAL_IPS
+|Yes
+|Yes
+
+|Enable external IPs in Network Grah via env_variable ROX_NETWORK_GRAPH_EXTERNAL_IPS
+|No
+|Yes
+|===
+
+
+* Enable external IPs collection per secured cluster runtime configuration: Each cluster can be individually configured to turn on collection of external IPs. To enable this feature on a given cluster, use “Collector Runtime configuration” (which is available from 4.6 on Tech. Preview) 
+
+=== secured-cluster-services:
+Refer to the runtime-configuration 
+
+runtime_config.yaml -> networking -> externalIps -> enable : true
+
+Note: It is supported to have clusters with the functionality enabled and others disabled at the same time. In this case, IP information will be available only for enabled clusters.
+
+
+* Enable external IPs in Central:  This is done using two environmental variables which are disabled by default: 
+- ROX_EXTERNAL_IPS enables storing external IP information in Central
+
+=== central-services: 
+With operator:
+
+	Insert the following customization in your Central CRD instance:
+
+----
+spec:
+  customize:
+    envVars:
+      - name: ROX_EXTERNAL_IPS
+        value: 'true'
+----
+
+With helm charts:
+
+update your installation with an overlay as follows:
+
+----
+customize:
+  # Extra environment variables for all containers in all objects.
+  envVars:
+    ROX_EXTERNAL_IPS: “true”
+----
+
+- ROX_NETWORK_GRAPH_EXTERNAL_IPS enables visualizing IPs in Network Graph.
+
+With operator:
+
+	Insert the following customization in your Central CRD instance:
+
+----
+spec:
+  customize:
+    envVars:
+      - name: ROX_NETWORK_GRAPH_EXTERNAL_IPS
+        value: 'true'
+----
+
+With helm charts:
+
+update your installation with an overlay as follows:
+
+----
+customize:
+  # Extra environment variables for all containers in all objects.
+  envVars:
+    ROX_NETWORK_GRAPH_EXTERNAL_IPS “true”
+----
+
+
+=== Querying External IP via API
+
+==== API endpoints
+
+Two API endpoints are provided for external IPs. The first is
+
+/v1/networkgraph/cluster/{clusterId}/externalentities
+
+Given a cluster ID, returns the list of external entities. Each entity consists of a name, CIDR block, if the entity is considered to be a default entity, and if the entity was discovered. In this context discovered means that the external IP address did not match any specified CIDR block. In this context default means that the external IP address does match one of the specified CIDR blocks.
```

Comment:
See my comment in https://docs.google.com/document/d/1iVBgbQ055OZQppRT--OXbyMt1kTw3IEA6dk2vkE7rgo/edit?disco=AAABUET_Aus

Reply:
I have changed the text here and in the google doc. I hop it is more accurate now.

Reply:
'default' entries are not external entities that 'match'; they are CIDR blocks defined by the system. Also, there is typo -> 'doesnot'